src/Goerli-DssSpell.t.base.sol

```diff
@@ -1505,10 +1505,7 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
             deployer == deployers.PE_CURRENT()
         ) ||
         (
-            deployer == deployers.ORACLES_1()
-        ) ||
-        (
-            deployer == deployers.ORACLES_2()
+            deployer == deployers.ORACLES()
```

Comment:
I think we still need the now `CES_05` address here otherwise the auth test is failing